### PR TITLE
Fix sizing of path to contrib image

### LINKED ISSUE #149: Illustration: Path to first analysis contribution
Create an illustration to show the steps outlined in #148

### PR DESCRIPTION
**Pull request by dvenprasad**
### Which issue does this address?
closes #149 

### Briefly describe the scope of the added docs file, including whether you link out to any external docs.
I split the image into two since increasing the font sizes did not do much for readability. 

Planning is now in one image. I added an extra step `Browse issues` since having only `pick an issue with no next step looked very odd. 

Implementation and Review are in the second one. They are together to preserve the cyclical nature of the two steps.

### Will you need additional visual aids for these docs?
no

### Author checklist

- [ ] The docs page has been added to the `nav` section in `mkdocs.yml`
- [ ] The docs adhere to the style guide [**structure directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)
- [ ] The docs adhere to the style guide [**language directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)
- [ ] The docs adhere to the style guide [**tone directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#tone)
- [X] The docs follow additional contributing instructions in [the docs contributing file](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md)

### Reviewer checklist

Please refer to the [docs style guide](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md) and the [docs contributing guide](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md) while reviewing this pull request.

- [ ] Do the docs adhere to the style guide [**structure directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)?
- [ ] Do the docs adhere to the style guide [**language directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#structure)?
- [ ] Do the docs adhere to the style guide [**tone directions**](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/general-style-guide.md#tone)?
- [ ] Do the docs follow additional contributing instructions in [the docs contributing file](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md)?

### CODE REVIEW

**Comment by sjspielman** [comment]
Just had a peek here, and there are a couple things we do need to address:

1. The images are not in the right place - they should be in `docs/img`, not `docs/getting-started/img`.
2. We prefer to use the formatting for images as explained here: https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md#adding-images

@dvenprasad can you make these changes before merging (you'll also need to update the relative link to the images once they are moved). Let me know if I can help!

**Comment by dvenprasad** [comment]
> We prefer to use the formatting for images as explained here: https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/docs/contributing.md#adding-images

I removed that because the alignment was a bit messy for the two images i.e., it was better for them to be aligned left w.r.t to each other and not center. But this is not a hill I want to die on. 

Remove that file. Not sure how that got there and nor is it being referenced. 

@sjspielman wanna take a quick look again?

**Comment by sjspielman** [comment]
> I removed that because the alignment was a bit messy for the two images i.e., it was better for them to be aligned left w.r.t to each other and not center. But this is not a hill I want to die on.

Ah, I see what you mean!!! I think you are right and should die on the hill...as it were....!

File: docs/getting-started/making-your-first-analysis-contribution.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
 # Making your first analysis contribution
 
+
+<figure markdown="span">
+
+      ![Path from joining to making a contribution- Planning](../img/first-analysis-contribution-planning.png){width="550"}
+
+</figure>
+
```
Comment (sjspielman) [inline docs/getting-started/making-your-first-analysis-contribution.md:9]:
```suggestion
<!-- These images intentionally do not use HTML formatting --> 

![Path from joining to making a contribution- Planning](../img/first-analysis-contribution-planning.png){width="550"}
```

File: docs/getting-started/making-your-first-analysis-contribution.md [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
 <figure markdown="span">
```
Comment (sjspielman) [inline docs/getting-started/making-your-first-analysis-contribution.md:10]:
```suggestion
```

File: docs/getting-started/making-your-first-analysis-contribution.md [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
-      ![Path from joining to making a contribution](../img/first-analysis-contribution.png){width="700"}
+
+      ![Path from joining to making a contribution- Implementation and Review](../img/first-analysis-contribution-implementation-review.png){width="700"}
+      
 </figure>
```
Comment (sjspielman) [inline docs/getting-started/making-your-first-analysis-contribution.md:14]:
```suggestion
![Path from joining to making a contribution- Implementation and Review](../img/first-analysis-contribution-implementation-review.png){width="700"}
```

**Review by sjspielman — APPROVED** [review]
👍

**Pull request merged**